packages/components/src/components/link-tile/link-tile.examples.md

```diff
@@ -236,6 +247,21 @@ export default class Code extends Vue {
 >
   <img src="${require('@/assets/image-grid.png')}" ${this.imgAttributes} />
 </p-link-tile>`};
+
+  hyphen = 'manual';
+  hyphens = ['auto', 'manual', 'none'];
+  get hyphenMarkup() {
+    return `<p-link-tile
+  href="https://www.porsche.com"
+  label="Some label"
+  description="An extra&shy;ordinarily Porsche"
```

Comment:
Should `&shy;` be kept around for all variants?
What about the cursor/tooltip behavior?

https://github.com/porsche-design-system/porsche-design-system/assets/4380295/6077a618-0d48-4b85-b478-52b0fbe27267

Reply:
I guess it makes sense to keep it in order to show when it has an effect.